app/services/transaction_search/date_range_searcher.rb

```diff
@@ -3,7 +3,7 @@ module TransactionSearch
   class DateRangeSearcher < BaseSearcher
 
     include DateHelper
-    FIELDS = %w(ordered_at fulfilled_at journal_or_statement_date).freeze
+    FIELDS = %w(ordered_at fulfilled_at reconciled_at journal_or_statement_date).freeze
```

Comment:
One other thing I thought of, but don't need to change right now is that it doesn't make sense to select reconciled or journal/statement while you're sending notifications. We're always using the same list, but a future enhancement might be to hide or disable the unusable options.